knowledge-base/actions/fabiankramm/release-asset-action/action-security.yml

```diff
@@ -0,0 +1,2 @@
+name: 'Release Asset'
+#No Permissions Needed
```

Comment:
There is `Content: write` permission to upload release asset. check [here](https://github.com/FabianKramm/release-asset-action/blob/6a964e8a644f28832c3f9a46abc69dda52fc6d17/main.js#L28)